app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -6,7 +6,10 @@ class Vmware < ComputeResource
     include ComputeResourceConsoleCommon
     include ComputeResourceCaching
 
-    validates :user, :password, :server, :datacenter, :presence => true
+    validates :user, :password, :datacenter, :presence => true
+    validates :server, :format => { :with => URI.regexp }, :presence => true,
+              :url_schema => ['http', 'https']
```

Comment:
Sorry for misleading you here, VMWare should /not/ have the `url_schema` validated. My mistake. Could you remove that please?